spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/edit.html

```diff
@@ -193,18 +194,26 @@ <h3> Select API user from table
                     responsive: true,
                     bFilter: false,
                     drawCallback: searchGituhbUsersCallBack,
-                    columns: [
-                        {data: "login"},
-                        {data: "name"},
-                        {data: "email"},
-                        {data: "avatar_url", render: function (data, type, row) {
+                    columns: initiateColumnsBasedOnTheIDMProvider([[${@environment.getProperty('ygg.admin.idm.provider')}]])
+                });
+
+                function initiateColumnsBasedOnTheIDMProvider(idmProvider) {
```

Comment:
this method is only called once and the parameter `idmProvider` is only used in one place, so does it make sense to pass it as parameter? Would probably be easier to understand if you do it like this:
```suggestion
                function initiateUserTableColumns() {
```
And in that method detect the IDM provider that is used.

Reply:
Done